build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
 
 plugins {
     id 'java'
+    id 'java-library'
```

Comment:
it seems funny to me that we need to build a java-library since this is a java service binary. what do we need this for?